### PR TITLE
Add save/load for pt2e example

### PR DESCRIPTION
**Pull request by Kaihui-intel**
## Type of Change

feature

## Description

detail description

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by yiliu30 — APPROVED** [review]
Please trigger some extension tests. Others are LGTM.

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_benchmark.sh [hunk 1]
```diff
@@ -4,17 +4,12 @@ set -x
 function main {
 
   init_params "$@"
-  run_benchmark
+  run_tuning
```
Comment (chensuyue) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_benchmark.sh:7]: Not the right place to change.

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/static_quant/pt2e/run_benchmark.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+# run_tuning
+function run_benchmark {
+
+    if [ "${topology}" = "opt_125m_pt2e_static" ]; then
+        model_name_or_path="facebook/opt-125m"
+        output_dir="saved_results"
+    fi
+    python run_clm_no_trainer.py --model ${model_name_or_path} --accuracy --output_dir ${output_dir} --tasks "lambada_openai"
```
Comment (chensuyue) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/static_quant/pt2e/run_benchmark.sh:44]: we should support accuracy/performance mode for pt2e right? not only accuracy. Like what we did for fx, https://github.com/intel/neural-compressor/blob/437c8e75706cff1767dcde115e428654766b3f18/examples/pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/fx/run_benchmark.sh#L28

File: examples/3.x_api/pytorch/cv/static_quant/run_quant.sh [hunk 3]
```diff
@@ -38,8 +38,9 @@ function init_params {
 function run_tuning {
```
Comment (chensuyue) [inline examples/3.x_api/pytorch/cv/static_quant/run_quant.sh:38]: Please add run_benchmark.sh for the benchmark test.

**Pull request merged**